cmd/_integration-tests/transport/handlers/server/server_handler.go

```diff
@@ -83,6 +83,11 @@ func (s transportService) GetWithPathParams(ctx context.Context, in *pb.GetWithQ
 	return &response, nil
 }
 
+// EchoOddNames implements Service. Why do we always put this comment here?
```

Comment:
We want to generate code that passes `golint` cleanly. The comment could be made more useful, but exported functions that do not have a documentation comment that begins with the function name will be flagged by golint.